index.html

```diff
@@ -2068,7 +2068,7 @@ <h3>Role Mapping Table</h3>
 				<tr id="role-map-time">
 					<th><a class="role-reference" href="#time"><code>time</code></a></th>
 					<td class="role-msaa-ia2">
-						<span class="property">Role: <code>ROLE_SYSTEM_STATICTEXT</code></span><br />
+						<span class="property">Role: <code>ROLE_SYSTEM_TEXT</code></span><br />
```

Comment:
I'd argue this should be ROLE_SYSTEM_GROUPING as well for the same reasons as caption. @aleventhal, what do you think?

Reply:
In Firefox, label has been mapped to ROLE_SYSTEM_STATICTEXT since at least 2012, probably much longer (I didn't go back further). Legend is mapped to label, so also ROLE_SYSTEM_STATICTEXT. I'm not sure if IE mapped to TEXT or STATICTEXT and I don't have a machine on which I can check. If password managers, etc. do rely on this, they would have been dealing with different mappings in different browsers for years.

I do follow your argument about STATICTEXT being meant for leaves, though I feel that mapping to TEXT is worse than STATICTEXT for clients.

Should we just change these to GROUPING as well?

Reply:
I made an issue for these other elements so I can land this PR: https://github.com/w3c/core-aam/issues/164

Reply:
I'm ok with keeping ROLE_SYSTEM_STATICTEXT for label/legend since this goes all the way back to IE, and keeping it the same is safest for backwards compat. I guess it doesn't need to be a leaf, whereas ROLE_SYSTEM_TEXT does. Being a static text means that it's a visible author-supplied label of some kind, apparently. A bit weird, but if it's not broken, let's not fix it :) @spectranaut , did Chrome expose static text for the same things as Firefox?